composer.json

```diff
@@ -8,8 +8,8 @@
         "composer-plugin-api":         "^1.0.0",
         "doctrine/annotations":        "^1.3",
         "phpdocumentor/type-resolver": "^0.2.0",
-        "symfony/filesystem":          "^2.3.0 | ^3.0.0",
-        "twig/twig":                   "1.*, >=1.13"
+        "symfony/filesystem":          "^2.3.0 || ^3.0.0",
+        "twig/twig":                   "^1.13|| ^2.0"
```

Comment:
minor detail, are we still putting the highest value first or does it not matter anymore? (more of a formatting thing)